docker-compose.yml

```diff
@@ -32,12 +32,10 @@ couchbase:
         - 11211
         - 18091
         - 18092
+    env_file: .env
     environment:
         - CONSUL_HOST=http://consul:8500
         - COUCHBASE_SERVICE_NAME=couchbase
-        - COUCHBASE_USER=Administrator
```

Comment:
You have a preference against the `- COUCHBASE_USER` syntax to inherit the env var, yes?